osgp/shared/osgp-ws-publiclighting/src/main/resources/schemas/pl-adhocmanagement.xsd

```diff
@@ -287,6 +259,46 @@
     </xsd:sequence>
   </xsd:complexType>
 
+  <xsd:complexType name="Status" />
+
+  <xsd:complexType name="DeviceStatus">
+    <xsd:annotation>
+      <xsd:documentation>
+        DeviceStatus represents the retrieved status of a smart device.
+      </xsd:documentation>
+    </xsd:annotation>
+    <xsd:complexContent>
+      <xsd:extension base="tns:Status">
+        <xsd:sequence>
+          <xsd:element maxOccurs="unbounded" name="LightValues" type="tns:LightValue" />
+          <xsd:element name="PreferredLinkType" type="tns:LinkType"
+            minOccurs="1" maxOccurs="1" />
+          <xsd:element name="ActualLinkType" type="tns:LinkType"
+            minOccurs="1" maxOccurs="1" />
+          <xsd:element name="LightType" type="tns:LightType"
+            minOccurs="0" maxOccurs="1" />
+          <xsd:element minOccurs="0" maxOccurs="8"
+            name="EventNotifications" type="tns:EventNotificationType" />
+        </xsd:sequence>
+      </xsd:extension>
+    </xsd:complexContent>
+  </xsd:complexType>
+
+  <xsd:complexType name="LightSensorStatus">
+    <xsd:annotation>
+      <xsd:documentation>
+        Light sensor status represents the retrieved status of a light measurement device.
+      </xsd:documentation>
+    </xsd:annotation>
+    <xsd:complexContent>
+      <xsd:extension base="tns:Status">
+        <xsd:sequence>
+          <xsd:element name="Status" type="tns:LightSensorStatusType" maxOccurs="1" />
```

Comment:
Attribute `maxOccurs` has its default value, like `minOccurs` that is left out here.
Probably best to include neither.